frost-core/src/frost/keys.rs

```diff
@@ -290,10 +298,6 @@ pub struct SharePackage<C: Ciphersuite> {
     pub identifier: Identifier<C>,
     /// This participant's secret share.
     pub secret_share: SecretShare<C>,
-    /// This participant's public key.
```

Comment:
We can address this deduping in a separate PR re this issue: https://github.com/ZcashFoundation/frost/issues/127